src/ScrollSync.js

```diff
@@ -9,6 +9,7 @@ import PropTypes from 'prop-types'
 export default class ScrollSync extends Component {
 
   static propTypes = {
+    onSync: PropTypes.func,
```

Comment:
Please add a JSDoc documentation for the function signature here using `/** */` comment. This should be displayed in the styleguide then (run it locally to verify).

Reply:
Done. Let me know if you want to change the description to something else and I'll update it right away.

I was trying to use JSDoc notations (e.g. `@param`) but they were not working in the styleguide.

Reply:
That's odd. It should work. See an example here: https://github.com/react-dropzone/react-dropzone/blob/master/src/index.js

Reply:
It may have to do with `styleguidist@5.0.0-beta8` being used (they're currently at version 11.0.8). I've tried upgrading but it's complaining about webpack 1 now.

Reply:
Let's do the JSDoc and I'll try upgrading the webpack etc when I have time in a separate PR (or you could as well)

Reply:
Docs updated 👍 

I wanted to upgrade `styleguidist` myself but I don't know when I'll have the time if it's not as simple as bumping it..

Reply:
Would be great if you could do that! Thanks!